webserver/observe/metrics/src/main/java/io/helidon/webserver/observe/metrics/JsonFormatter.java

```diff
@@ -471,6 +475,42 @@ protected void apply(JsonObjectBuilder builder) {
                 builder.add(meterId.name(), sameNameBuilder);
             }
 
+            private void addDetails(HistogramSnapshot snapshot, Meter.Id childId, TimeUnit timeUnit) {
+                snapshot.percentileValues().forEach(vap ->
+                                                            sameNameBuilder
```

Comment:
IMO, all these new lines don't make this code fragment more readable. I would make this code a bit less sparse.